setup.py

```diff
@@ -36,16 +36,16 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 setup(name='mbed-flasher',
-      version='0.6.1',
+      version=VERSION,
       description=DESCRIPTION,
       long_description=read('README.md'),
       author=OWNER_NAMES,
       author_email=OWNER_EMAILS,
       maintainer=OWNER_NAMES,
       maintainer_email=OWNER_EMAILS,
       url='https://github.com/ARMmbed/mbed-flasher',
-      packages=find_packages(),
-      package_data={'': ['FlasherMbed.target_info.json']},
+      download_url= 'https://github.com/ARMmbed/mbed-flasher/tarball/'+VERSION,
```

Comment:
consider
```
"https://github.com/ARMmbed/mbed-flasher/tarball/{}".format(VERSION)
```

Reply:
oh, and now that you say that, I noticed that the url is wrong....:D
https://api.github.com/repos/ARMmbed/mbed-flasher/tarball/v0.6.1

and you need to have the v in the version

Reply:
fixed